byway/src/main/webapp/destinations.js

```diff
@@ -61,8 +61,7 @@ function initAutocomplete() {
   const START_SEARCH_BOX = 'start-search-box';
   const DESTINATIONS_SEARCH_BOX= 'destinations-search-box';
```

Comment:
Doesn't seem like you use these constants elsewhere, so rather than create these consider inputting the string directly into `createSearchBox()`.

Reply:
done